### ISSUE TITLE
stickybox: failing to use it

### ISSUE THREAD

**Issue by drupol** [lkoehl/typst-boxes#6]
Hi,

I'm trying to use a stickbox and I have this issue:

```
error: cannot divide ratio by length
    ┌─ @preview/colorful-boxes:1.2.0/lib.typ:159:28
    │
159 │           width: calc.clamp(width * 0.35 / 1cm, 1, 4) * 1cm,
    │                             ^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `stickybox`
   ┌─ src/doc/main.typ:95:2
   │
95 │   stickybox(rotation: 30deg, content)
   │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

help: error occurred in this call of function `default-rect`
    ┌─ @preview/drafting:0.1.2/drafting.typ:288:18
    │  
288 │     let note-rect = props.at("rect")(
    │ ╭───────────────────^
289 │ │     stroke: props.stroke, width: right-width, body
290 │ │   )
    │ ╰───^

help: error occurred in this call of function `_margin-note-right`
    ┌─ @preview/drafting:0.1.2/drafting.typ:368:4
    │  
368 │ ╭     margin-func(
369 │ │       body, dy, anchor-x, anchor-y, ..properties
370 │ │     )
    │ ╰─────^

help: error occurred in this function call
   ┌─ @preview/drafting:0.1.2/drafting.typ:49:4
   │
49 │     func(use-loc)
   │     ^^^^^^^^^^^^^
```

**Comment by lkoehl** [comment]
Could you please share your code, which produces this error?
The following code works just fine for me:
```
#stickybox(
  rotation: 3deg,
  width: 6cm
)[
  #lorem(20)
]
```

**Comment by lkoehl** [comment]
Calculating the width when using ratios caused wrong dimensions. The stickybox could only be used with absolute widths e.g. 5cm. Should be fixed in version 1.3.1.
